packages/components/src/components/SquigglePlayground/index.tsx

```diff
@@ -234,10 +224,14 @@ export const SquigglePlayground: React.FC<PlaygroundProps> = (props) => {
       : `${executionTime}ms`;
 
   const playgroundWithEditor = (
-    <div className="flex flex-row">
+    <div className="flex h-full items-stretch">
       <ResizableBox
-        className="h-full relative"
-        width={initialWidth / 2}
+        className={clsx("relative", !initialWidth && "w-1/2")}
+        width={
+          initialWidth === undefined
+            ? (null as any) // we intentionally pass the invalid value to ResizableBox when initialWidth is not set yet
+            : initialWidth / 2
```

Comment:
The above lines are a bit confusing.

Part of me wants this, but it's even more wordy.
```ts
  withInitialWidth = { className: "", width: initialWidth/2 }
  withoutInitialWidth = { className: "w-1/2", width: null as any }
 ...
  className={clsx("relative", !!initialWidth ? withInitialWidth.className : withoutInitialWidth.className)}
  ```